data/io.elementary.calendar.appdata.xml.in

```diff
@@ -13,13 +13,18 @@
     </p>
   </description>
   <releases>
-    <release version="5.0.7" date="2020-06-30" urgency="medium">
+    <release version="5.1.0" date="2020-06-30" urgency="medium">
       <description>
+      <p>Improvements:</p>
+      <ul>
+        <li>Allow addition and removal of recurrence to existing event</li>
```

Comment:
```suggestion
        <li>Allow addition and removal of recurrences to existing event</li>
```
Shouldn't this be plural?

Reply:
Strictly speaking it should be "Allow addition and removal of recurrences to and from existing events" (You cannot remove something to something); but that is ugly.

Perhaps "Allow adding and removing recurrences in existing events"?

Reply:
Sure, that sounds good. Certainly good enough for a little sentence in the release notes :slightly_smiling_face:

Reply:
On reflection I think I'll just remove this line - the other line pretty much covers what this PR does from a user viewpoint.

Reply:
Done